sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -2141,7 +2141,7 @@ OUT
 limitWeeklyChildCare
 limitWeeklyAlimonyChildSupport
 limitWeeklySecondResidence
-limitWeeklytransportationAllownace
```

Comment:
Is it about fixing the comment here ? - If so I would encourage to stay away from it as we have to fix those one in many places in camunda and also save dev time on looking into what is it.

![image](https://github.com/bcgov/SIMS/assets/54600590/db00c2b0-26db-481e-aff2-ab72d61d390c)

Reply:
Sounds good to me. I've reverted to the original files in my latest commit.

Reply:
Thanks and having said that I am fine with PT* files as you are already modifying it